app/src/main/java/com/poupa/vinylmusicplayer/helper/PrevNextButtonOnTouchListener.java

```diff
@@ -0,0 +1,63 @@
+package com.poupa.vinylmusicplayer.helper;
+
+import android.view.MotionEvent;
+import android.view.View;
+import android.os.Handler;
+import android.view.View.OnClickListener;
+
+public class PrevNextButtonOnTouchListener implements View.OnTouchListener {
+    private Handler handler = new Handler();
+
+    private int initialInterval;
+    private final int normalInterval;
```

Comment:
Can you please rename these variables to make the usage explicit (i.e. skipTriggerIntervalMillis)

Reply:
Resolved in the most recent commit